routing/nearest_edge_finder.hpp

```diff
@@ -25,8 +26,10 @@ class NearestEdgeFinder
 {
   struct Candidate
   {
-    double m_dist = std::numeric_limits<double>::max();
-    uint32_t m_segId = 0;
+    static uint32_t constexpr kInvalidSegmentId = std::numeric_limits<uint32_t>::max();
```

Comment:
nit: s/uint32_t/auto/ as it's clear from ```std::numeric_limits<uint32_t>::max()```

Reply:
Done